flang/runtime/stop.cpp

```diff
@@ -152,11 +157,34 @@ void RTNAME(PauseStatementText)(const char *code, std::size_t length) {
   std::exit(status);
 }
 
+static void PrintBacktrace() {
+#ifdef HAVE_BACKTRACE
+  // TODO: Need to parse DWARF information to print function line numbers
+  constexpr int MAX_CALL_STACK{999};
+  void *buffer[MAX_CALL_STACK];
+  int nptrs{backtrace(buffer, MAX_CALL_STACK)};
+
+  if (char **symbols{backtrace_symbols(buffer, nptrs)}) {
+    for (int i = 0; i < nptrs; i++) {
+      Fortran::runtime::Terminator{}.PrintCrashArgs("#%d %s\n", i, symbols[i]);
+    }
+    free(symbols);
+  }
+
+#else
+
+  // TODO: Windows platform implementation
```

Comment:
Is this comment still accurate? Should it instead say "Handle the case when a backtrace is not available", or something along those lines? 

Does it makes sense to print a message saying that a backtrace was not available instead? It probably does not matter in the case of `abort`, but if the non-standard `backtrace` extension is used, it may be confusing if, it doesn't print anything on some platform".

Reply:
I think you are right, doing nothing does lead to confusion, I will add this, thanks for the suggestion

Reply:
I have modified this point. Thank you again for your suggestion. It is really necessary.